### ISSUE TITLE
No “tracking” option in the simba tool.

### ISSUE THREAD

**Issue by klxxxx** [sgoldenlab/simba#240]
**Describe the bug**
I installed SimBA using anaconda, but after I started Simba, there was no tracking option in the toolbar, as shown in the image.

**My version.**
![b11fca32c0635e05c0399f4d98c485e](https://user-images.githubusercontent.com/89234825/227890409-9f82e9c2-4beb-4060-947b-e4699b025c86.png)

**Reference version.**

![1402f030e13e09c7059fa32a480c2ab](https://user-images.githubusercontent.com/89234825/227890444-ac7a1168-d1a7-4c11-8ad4-971c4b12f6dd.png)

**Comment by sronilsson** [comment]
Hi @klxxxx ! Although the drop-down menu was there in earlier versions, the menu items where not tied to any function - you most likely would have hit error if you clicked them. We stopped supporting the tracking tools some time ago. Mainly because I answered a lot of troubleshooting questions about these packages and not SimBA. I recommend using the packages through their respective interfaces instead of through SimBA.
